app-backend/tileserver/src/main/scala/Routes.scala

```diff
@@ -0,0 +1,74 @@
+package com.azavea.rf.tileserver
+
+import akka.http.scaladsl.model.{ContentType, HttpEntity, HttpResponse, MediaTypes}
+import com.azavea.rf.tileserver.image._
+import com.azavea.rf.tileserver.image.ImageParams._
+import com.azavea.rf.tileserver.model._
+import com.azavea.rf.tileserver.model.ModelParams._
+import geotrellis.raster._
+import geotrellis.raster.histogram.Histogram
+import geotrellis.raster.render.{ColorRamp, ColorMap}
+import geotrellis.spark._
+import akka.http.scaladsl.server._
+import akka.http.scaladsl.server.Directives._
+import com.typesafe.scalalogging.LazyLogging
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent._
+
+
+trait Routes extends LazyLogging {
+
+  def singleLayer: Route =
+    pathPrefix(Segment / Segment / JavaUUID / Segment / IntNumber / IntNumber / IntNumber) { (bucket, prefix, layerId, mode, zoom, x, y) =>
```

Comment:
Bucket should be provided via the `config` and from the environment.

What does prefix represent in this case and layerId?